### PR TITLE
add GR105 - validate no duplicate content item exist

### PR DESCRIPTION
**Pull request by ilaner**
## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5947?filter=-4&jql=resolution%20%3D%20Unresolved%20AND%20labels%20%3D%20sdk-validate%20order%20by%20created%20DESC

### CODE REVIEW

File: demisto_sdk/commands/content_graph/interface/neo4j/neo4j_graph.py [hunk 1]
```diff
@@ -128,7 +129,7 @@ def __init__(
             if artifacts_folder := os.getenv("ARTIFACTS_FOLDER"):
                 output_path = Path(artifacts_folder) / "content_graph"
                 output_path.mkdir(parents=True, exist_ok=True)
-            update_content_graph(self, use_git=True, output_path=output_path)
+            update_content_graph(self, use_current=True, output_path=output_path)
```
Comment (dorschw) [inline demisto_sdk/commands/content_graph/interface/neo4j/neo4j_graph.py:132]:
```suggestion
            update_content_graph(self, use_git=True, output_path=output_path)
```

File: demisto_sdk/commands/content_graph/interface/neo4j/neo4j_graph.py [hunk 2]
```diff
@@ -361,6 +362,22 @@ def get_duplicate_pack_display_name(
...
+            all_nodes = []
+            for content_item, dups in duplicates:
+                all_nodes.append(content_item)
+                all_nodes.extend(dups)
+            self._add_nodes_to_mapping(all_nodes)
+            duplicate_models = []
+            for content_item, dups in duplicates:
+                dups = [self._id_to_obj[duplicate.id] for duplicate in dups]
+                duplicate_models.append((self._id_to_obj[content_item.id], dups))
+            return duplicate_models
```
Comment (dorschw) [inline demisto_sdk/commands/content_graph/interface/neo4j/neo4j_graph.py:379]: can we unindent this? or do we still need the session here?

File: demisto_sdk/commands/content_graph/tests/graph_validator_test.py [hunk 3]
```diff
@@ -578,3 +588,21 @@ def test_validate_dependencies(repository: ContentDTO, caplog, mocker):
         logger_info.call_args_list,
         "The core pack SamplePack cannot depend on non-core packs: ",
     )
+
+
+def test_validate_duplicate_id(repository: ContentDTO, capsys):
+    """
+    Given
+    - A content repo
```
Comment (dorschw) [inline demisto_sdk/commands/content_graph/tests/graph_validator_test.py:596]: please mention all prerequisites - the fact that there inherently is a dupe etc

**Review by dantavori — COMMENTED** [review]
looks good to me, check the test failure
I recommend checking the validation in a gitlab content build (although it's tested in the circle build here), if something is wrong it can make noise

**Pull request merged**